go.mod

```diff
@@ -130,7 +130,7 @@ require (
 	k8s.io/cri-api v1.20.0
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v1.20.5
```

Comment:
The submodules here got incorrectly versioned at some point. Except for the main kubernetes module and klog/yaml/client-go, the ones that are currently at v1.20.5 or v0.20.5 should move to v0.20.11. See the other branches for an example.

Reply:
I think now they are all on 1.20.11